src/Tribe/Views/V2/iCalendar/Links/Link_Interface.php

```diff
@@ -68,7 +68,7 @@ public function set_visibility( bool $visible );
 	 *
 	 * @return string The translated link text/label.
 	 */
-	public static function get_label( \Tribe\Events\Views\V2\View $view );
+	public function get_label( \Tribe\Events\Views\V2\View $view );
```

Comment:
This was existing, but can you  add use \Tribe\Events\Views\V2\View;
To the top of the file and change these to: View

Reply:
Thanks!
Yeah, I'll change it in the signature, but I think it best to leave it in the docblock - since those can be viewed out of context.